### DISCUSSION TITLE
Multiple Values in Archive Field

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#14045] [Archive]
**When you create an archive Field, you are prompted to specify an "archive" value and an "unarchived" value:**

<img width="1022" alt="Screen Shot 2022-06-22 at 2 00 54 PM" src="https://user-images.githubusercontent.com/62306630/175116183-72cf89e9-f950-46f6-860c-68730df7de3f.png">

---

**However, your archive Field can contain other values as well**

<img width="1050" alt="Screen Shot 2022-06-22 at 2 02 05 PM" src="https://user-images.githubusercontent.com/62306630/175116199-04a4e199-319e-40df-92ee-799a3b5e4b3f.png">

----

I also noticed that these "other values" seem to behave just the same as the specified "unarchived value".
I'm wondering if there is any difference I haven't detected?
I can't find this in the existing discussions *(however, there's a lot, so I may have missed it. If so, sorry gang!)*

**Comment by rijkvanzanten** [comment]
The unarchive value field controls what value the field is set to when you click the unarchive button, not what values are considered "Unarchived" 🙂
